### PR TITLE
[16.0][IMP] shopinvader_schema_sale: sale_line qty rounding

### PR DESCRIPTION
**Pull request by qgroulard**

### CODE REVIEW

**Review by lmignon — APPROVED** [review]

**Review by benwillig — APPROVED** [review]

File: shopinvader_schema_sale/schemas/sale_line.py [hunk 1]
```diff
@@ -20,5 +22,8 @@ def from_sale_order_line(cls, odoo_rec):
             product_id=odoo_rec.product_id.id,
             name=odoo_rec.name,
             amount=SaleLineAmount.from_sale_order_line(odoo_rec),
-            qty=odoo_rec.product_uom_qty,
+            qty=float_round(
```
Comment (lmignon) [inline shopinvader_schema_sale/schemas/sale_line.py]: you must use `json_float_round`

**Review by lmignon — CHANGES_REQUESTED** [review]

File: shopinvader_schema_sale/schemas/sale_line.py (hunk 1 above)
Comment (qgroulard) [reply shopinvader_schema_sale/schemas/sale_line.py]: Done

**Review by lmignon — APPROVED** [review]

**Pull request merged**